skimage/feature/brief.py

```diff
@@ -112,7 +118,7 @@ class BRIEF(DescriptorExtractor):
     """
 
     def __init__(self, descriptor_size=256, patch_size=49,
-                 mode='normal', sigma=1, sample_seed=1):
+                 mode='normal', sigma=1, sample_seed=None):
```

Comment:
why change the default here?

Reply:
If we leave it `sample_seed=1` I think this would give different random values than with the legacy `RandomState(1)`, but it would at least still be the same values each time it is run?

Reply:
I would leave the decision to you here as I don't have any background here 😅. I just checked the doctests we're passing here, which is the case. In general it's not a good idea to have seeds set in stones but here it might be a special case.

Reply:
Let's see what others think. I have never used the feature detection module much myself.

Reply:
Yeah tbh this would be a breaking change, I agree that it should not have been 1 to begin with but now we are sort of stuck with it. I'd revert it, and leave the decision of what to do with it to a later issue.

Reply:
I reverted to 1 then.